### PR TITLE
Deprecations and Removals for Chrome 74

### PR DESCRIPTION
**Pull request by jpmedley**
**Target Live Date:** 2019-03-21

- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by jpmedley** [comment]
@petele I will be boarding a plane when beta goes live tomorrow. Can you please push this for me?

**Comment by jpmedley** [comment]
I mean merge. Letting the auto system push it is fine.

File: src/content/en/updates/2019/03/chrome-74-deps-rems.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+[Intent to Remove](https://crbug.com/844455) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5989473649164288) &#124;
+[Chromium Bug](https://groups.google.com/a/chromium.org/d/topic/blink-dev/MkA0A1YKSw4/discussion)
+
+## Deprecate drive-by downloads in sandboxed iframes
+
+Chrome will soon prevent downloads in sandboxed `iframes` that lack a user
+gesture, though this restriction could be lifted via an 
+`allow-downloads-without-user-activation` keyword in the sandbox attribute list.
```
Comment (petele) [inline src/content/en/updates/2019/03/chrome-74-deps-rems.md]: @jpmedley - I suspect this should be on a single line? There was a line break in the middle. I changed it, but please confirm.

Comment (jpmedley) [reply src/content/en/updates/2019/03/chrome-74-deps-rems.md]: Correct

**Pull request merged**